spec/integration/source_command_spec.rb

```diff
@@ -13,6 +13,18 @@
         expect(gitsh).to output /Yes it did!/
       end
     end
+
+    it 'expands ~ in paths' do
+      GitshRunner.interactive do |gitsh|
+        write_file("#{ENV['HOME']}/.gitshrc", ':set source_worked "True"')
```

Comment:
Maybe, but I quite like how explicit this is: it means the test makes it clear what `~` should expand to.